Tests/Marketplace/upload_packs_private.py

```diff
@@ -234,25 +236,25 @@ def clean_non_existing_packs(index_folder_path, private_packs, storage_bucket):
 
     if invalid_packs_names:
         try:
-            print_warning(f"Detected {len(invalid_packs_names)} non existing pack inside index, starting cleanup.")
+            logging.warning(f"Detected {len(invalid_packs_names)} non existing pack inside index, starting cleanup.")
 
             for invalid_pack in invalid_packs_names:
                 invalid_pack_name = invalid_pack[0]
                 invalid_pack_path = invalid_pack[1]
                 # remove pack from index
                 shutil.rmtree(invalid_pack_path)
-                print_warning(f"Deleted {invalid_pack_name} pack from {GCPConfig.INDEX_NAME} folder")
+                logging.warning(f"Deleted {invalid_pack_name} pack from {GCPConfig.INDEX_NAME} folder")
                 # important to add trailing slash at the end of path in order to avoid packs with same prefix
                 invalid_pack_gcs_path = os.path.join(GCPConfig.STORAGE_BASE_PATH, invalid_pack_name, "")  # by design
 
                 for invalid_blob in [b for b in storage_bucket.list_blobs(prefix=invalid_pack_gcs_path)]:
-                    print_warning(f"Deleted invalid {invalid_pack_name} pack under url {invalid_blob.public_url}")
+                    logging.warning(f"Deleted invalid {invalid_pack_name} pack under url {invalid_blob.public_url}")
                     invalid_blob.delete()  # delete invalid pack in gcs
-        except Exception as e:
-            print_error(f"Failed to cleanup non existing packs. Additional info:\n {e}")
+        except Exception:
+            logging.exception("Failed to cleanup non existing packs.")
```

Comment:
the exception is very broad, are you sure you want to remove the print of it?
you can add it in debug.

relevant for all exceptions

Reply:
When you use logging exception in an exception handling context it automatically print the trace and the message of the exception free of charge with ERROR level. 
There is no need to add the print as it's automatically printed

![image](https://user-images.githubusercontent.com/41257953/97853687-20f48280-1d01-11eb-92dd-0fe9a1fa1fef.png)